test/e2e/scheduling/predicates.go

```diff
@@ -491,11 +491,14 @@ var _ = SIGDescribe("SchedulerPredicates [Serial]", func() {
 		framework.ExpectEqual(labelPod.Spec.NodeName, nodeName)
 	})
 
-	// 1. Run a pod to get an available node, then delete the pod
-	// 2. Taint the node with a random taint
-	// 3. Try to relaunch the pod with tolerations tolerate the taints on node,
-	// and the pod's nodeName specified to the name of node found in step 1
-	ginkgo.It("validates that taints-tolerations is respected if matching", func() {
+	/*
+		Release : v1.17
+		Testname: Scheduler, taints-tolerations matching
+		Description: After removing the pod form the available node, verify that the
+		pod (with toleration against node taint) is scheduled when the node is tainted with a
+		random taint. The pod MUST be scheduled onto the node
+	*/
```

Comment:
The wording here is a bit unclear to me.

The basic idea here is: Find an available node which can afford the testing pod, and taint that node with a specific key with effect NoSchedule, then schedule a Pod with (1) particular nodeLabel to ensure it can only be placed onto node, and also (2) particular tolerations spec to ensure it matches the pre-applied taint. Finally check if the Pod is running on the node.

And also, L529~L533 may be a copy/paste mistake, we need to reword them as well.

Could you reword these 2 places?

Reply:
ack, reworded the description. Sorry what is the issue with L529-533? It seems to be doing the same checks as the tests above it?

Reply:
Nvm, L529~L533 is fine. Just leave them as is.

Reply:
Just did a rebase, would you mind reviewing?